### DISCUSSION TITLE
The need to use media priors - Why is it required?

### DISCUSSION THREAD

**Discussion by AruunnAnanda** [google/lightweight_mmm#121] [Q&A] [answered]
Hello there,

The intention is to understand why we need media priors and how does it help the model? Can someone please help me understand it?

After reading some online documentation I understood we only need priors for unknown parameters like **slope** and **intercept** because that is what we're trying to solve. 

Now, it confuses me a bit when I think of having priors for media budgets. 

Thanks!

**Comment by michevan** [comment, marked as answer]
So the (scaled) media budgets are used as the priors for the media channel coefficients, we don't have priors for the media budgets.

The papers in the references section on the main Github page (particularly [this one](https://research.google/pubs/pub46001/)) might help this make more sense!

**Comment by AruunnAnanda** [comment]
Hi @michevan,

Thanks for sharing the documentation link. 

Just for clarification - I was referring to the highlighted argument in the screenshot. Even you're talking about the same, right?

<img width="750" alt="Screenshot 2022-12-15 at 09 19 10" src="https://user-images.githubusercontent.com/107944288/207809983-b7435ffb-bb08-4d99-ba20-a12e6807908d.png">

**Comment by michevan** [comment]
yep!

**Reply by AruunnAnanda** [reply]
Thanks!
